doc/source/security/encrypting-threads.rst

```diff
@@ -93,4 +93,16 @@ SQL statement.
 
     :ref:`using-keyring-plugin`
 
+.. rubric:: Upgrading from version 5.7 
+
+When upgrading from version 5.7, before you turn on encryption, you should
+consult INFORMATION_SCHEMA_ENCRYPTION to verify which tables are excluded. If
+you wish to exclude a table, use ``ALTER TABLE ... ENCRYPTION='N'``. 
+
+If a table has ``ENCRYPTION='Y'``, the table stays encrypted. A table that is
+not encrypted remains not encrypted. 
+
+During an upgrade, all non-encrypted tables are set to ``ENCRYPTION='N'``
+without regard for the database encryption.
```

Comment:
This is called implicit ENCRYPTION='N' and does not cause tablespace to be excluded from encryption threads. Only explicit ENCRYPTION='N' causes tablespace to be excluded from encryption threads. This might be bit confusing - so the best way to check which tablespaces are excluded from encryption threads is to check INFORMATION_SCHEMA.INNODB_TABLESPACES_ENCRYPTION. If the table/tablespace you want to excluded is not in this table (and have field EXCLUDED set to 'Y'), issue ALTER TABLE/TABLESPACE ENCRYPTION='N'.

Reply:
Updated.

Reply:
The upgrade scenario between 5.7 and 8.0 should be removed for encryption threads. This is because we have decided to remove encryption threads from 5.7.

Reply:
Removed.